test/pytest/test_handler.py

```diff
@@ -1,46 +1,60 @@
-import os
```

Comment:
I don't believe this file will run, it needs an `import subprocess`

Might just be worth typing `python test_handler.py` to see if it compiles or not